monosi/monitors/base.py

```diff
@@ -15,19 +18,52 @@ class Schedule(DataClassDictMixin):
     minutes: int = 720
     type: ScheduleType = ScheduleType.INTERVAL
 
+def temp_transform_to_tests(metric_stats) -> List[AnomalyDetectorTest]:
+    tests = []
+
+    for column in metric_stats.keys():
+        for metric in metric_stats[column].keys():
+            values = metric_stats[column][metric]
+
+            values: List[float] = list(filter(lambda x: x == None, values))
```

Comment:
why do this here if we already do it on the `__init__` method of the `AnomalyDetectorTest` class

Reply:
This is the wrong question, but it's a great catch.

This /should/ be passing stats instead of values based on the method signature cause it's forward looking to needing to access the stats, but we actually shouldn't need to.

AKA it should stay here because actually Tests shouldn't filter out None (this is obvious when we think about a null rate test), but we are passing values as the wrong object ('stats') and we should change that in AnomalyDetectorTest.